esmvaltool/cmorizers/data/formatters/datasets/esrl.py

```diff
@@ -13,23 +13,22 @@
 
```

Comment:
When running `esmvaltool data format ESRL` it seems to me that only the NCL script is called, not the Python one.

Reply:
Both should be run or the python one is the replacement for the NCL?

Reply:
Just checked in main, The NCL is no longer there so it is a replacement. Now the NCL is removed